README.md

```diff
@@ -1,2 +1,71 @@
-# vue-cli-plugin-vuedock
-Docker plugin for the VueCLI
+# VueDock
+> Docker plugin for the Vue CLI 3
+
+[![Build Status](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock.svg?branch=master)](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock)
+
```

Comment:
```suggestion

Generates docker and compose files for both development and hosting, allowing:
- running the development server
- running unit tests
- running nightwatch e2e tests
- hosting built application with nginx container

## Installation in a Vue project
Simply run from the project root:
```vue add vuedock```

For more information, see: https://cli.vuejs.org/guide/plugins-and-presets.html#installing-plugins-in-an-existing-project
```

Reply:
Ah never knew you could do this in github, very cool.